x-pack/elastic-agent/docs/inputs/input-logfile.asciidoc

```diff
@@ -0,0 +1,163 @@
+:type: logfile
```

Comment:
Not sure this is required for the new docs (depends on how you plan to handle the includes). I'm trying to keep the files as simple as possible going forward.

Reply:
So you would just remove it?